pkg/slayers/path/hopfield_spec.gobra

```diff
@@ -16,6 +16,32 @@
 
 package path
 
+ghost const MetaLen = 4
+
 pred (h *HopField) Mem() {
 	acc(h) && h.ConsIngress >= 0 && h.ConsEgress >= 0
 }
+
+ghost 
+decreases
+pure func infoFieldOffset(currINF int) int {
```

Comment:
If this is meant to be used outside of the package, then we should make it public. Even though Gobra does not enforce this now, it will likely do so in the future

Reply:
In this case it is meant to be private.